Examples/CMakeLists.txt

```diff
@@ -1 +1,73 @@
-add_subdirectory (CppClient)
+set (
+  CARLA_EXAMPLE_CPP_CLIENT_SOURCE_PATH
+  ${CARLA_WORKSPACE_PATH}/Examples/CppClient
+)
+
+set (
+  CARLA_EXAMPLE_CPP_DEPENDENCIES
+  carla-client
+  rpc
+  png_static
+  zlibstatic
+  Boost::asio
+  Boost::python
+  Boost::geometry
+  Boost::gil
+  RecastNavigation::Recast
+  RecastNavigation::Detour
+  RecastNavigation::DetourCrowd
+)
+
+if (BUILD_EXAMPLES)
+  carla_message ("CARLA Building C++ EXAMPLE CLIENT")
+
+  project (
+    carla-example-cpp-client
+    LANGUAGES
+      CXX
+    VERSION
+      ${CARLA_VERSION}
+  )
+
+  file (
+    GLOB_RECURSE 
+    CARLA_EXAMPLE_CPP_SOURCES
+    CONFIGURE_DEPENDS 
+    ${CARLA_EXAMPLE_CPP_CLIENT_SOURCE_PATH}/*.cpp
+  )
+
+  file (
```

Comment:
CARLA_EXAMPLE_CPP_HEADERS is not referenced in carla_add_executable.

Reply:
Updated